packages/html-ejs/package.json

```diff
@@ -13,7 +13,7 @@
     "test": "mocha --no-config --require ts-node/register src/index.test.ts"
   },
   "dependencies": {
-    "@prairielearn/html": "^3.0.0",
+    "@prairielearn/html": "workspace:^",
     "ejs": "^3.1.9"
   },
   "devDependencies": {
```

Comment:
The dev dependency for tsconfig wasn't updated here.

Reply:
That doesn't matter as much since it's just a dev dependency, but I'll change it anyways for consistency.